nodes/nomos-node/src/api.rs

```diff
@@ -124,6 +182,11 @@ where
                 "/cryptarchia/headers",
                 routing::get(cryptarchia_headers::<T, S, SIZE>),
             )
+            .route("/da/add_blob", routing::post(add_blob::<A, B, VB, S>))
+            .route(
+                "/da/get_range",
+                routing::post(get_range::<T, C, V, VP, S, SIZE>),
+            )
```

Comment:
```suggestion
            .route(
                "/da/get_range",
                routing::get(get_range::<T, C, V, VP, S, SIZE>),
            )
```
Does this need to be `get`? (and in the related code below)

Reply:
Yes, `GET` here would be more idiomatic. I'm not sure if it needs to be :) Will update the app_id and range as query params.

Reply:
Yeah. I don't think it's very important. Just asked to check if it was a typo or not. I'm fine with either.